$NetBSD: patch-lib_Makefile,v 1.6 2018/07/03 14:27:59 adam Exp $

Enable the "install" target on all OS.
Fix pkgconfig installation path.

--- lib/Makefile.orig	2018-06-28 08:55:26.000000000 +0000
+++ lib/Makefile
@@ -154,7 +154,6 @@ clean:
 #-----------------------------------------------------------------------------
 # make install is validated only for Linux, macOS, BSD, Hurd and Solaris targets
 #-----------------------------------------------------------------------------
-ifneq (,$(filter $(shell uname),Linux Darwin GNU/kFreeBSD GNU OpenBSD FreeBSD NetBSD DragonFly SunOS))
 
 DESTDIR     ?=
 # directory variables : GNU conventions prefer lowercase
@@ -168,11 +167,7 @@ LIBDIR      ?= $(libdir)
 includedir  ?= $(PREFIX)/include
 INCLUDEDIR  ?= $(includedir)
 
-ifneq (,$(filter $(shell uname),FreeBSD NetBSD DragonFly))
-PKGCONFIGDIR ?= $(PREFIX)/libdata/pkgconfig
-else
 PKGCONFIGDIR ?= $(LIBDIR)/pkgconfig
-endif
 
 ifneq (,$(filter $(shell uname),SunOS))
 INSTALL ?= ginstall
@@ -232,4 +227,3 @@ uninstall:
 	@$(RM) $(DESTDIR)$(INCLUDEDIR)/zdict.h
 	@echo zstd libraries successfully uninstalled
 
-endif
